src/components/periodic-table/PeriodicTable.test.tsx

```diff
@@ -19,29 +19,27 @@ function elementRenderer(atomic: number): IPeriodicTableElement {
   };
 }
 
-async function renderTable() {
-  render(<PeriodicTable elementRenderer={elementRenderer} />);
-  await waitForElementToBeRemoved(
-    () => screen.queryAllByLabelText(/loading/i),
-    { timeout: 4000 }
-  );
-}
-
 describe("should render the periodic table", () => {
   beforeAll(() => {
     ElementManager.loadElements();
```

Comment:
Is there a reason we need this, is the component itself not calling that getter?

Reply:
The timeout?

Reply:
` ElementManager.loadElements();`

Reply:
Yes. The PeriodicTable is being rendered inside PeriodicTablePage, which is responsible for loading the data and providing them to the component.
So, I just called the getter to sort of mock the behaviour.

Reply:
We've mentioned in other threads that perhaps this load should be done in `setupTests` as quite a few components depend on it.